sdk/storage/azure-storage-blob-cryptography/README.md

```diff
@@ -100,8 +104,6 @@ Blob storage is designed for:
 - Storing data for backup and restore, disaster recovery, and archiving.
 - Storing data for analysis by an on-premises or Azure-hosted service.
 
-## Examples
```

Comment:
Even though this section is blank there is validation in place that checks that README's include this header.

Reply:
I have added the header `##Example`